app/marketing/mails.py

```diff
@@ -479,6 +497,28 @@ def new_quest_request(quest, is_edit):
         translation.activate(cur_language)
 
 
+def new_action_request(action):
+    to_email = settings.PERSONAL_CONTACT_EMAIL
```

Comment:
We should definitely get a template for this ASAP - I get some complaints that when our email notifications are too simple they look suspicious

Reply:
in general i'd agree; but this is an admin only email :P

Reply:
ah true - but sometimes we let janky emails slip out! :) something to stay mindful of